Samples/src/FSharpWithQSharp/QuantumCode/Operations.qs

```diff
@@ -0,0 +1,77 @@
+﻿namespace QuantumCode
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+
+    // The oracle: f(x₀, …, xₙ₋₁) = Σᵢ (rᵢ xᵢ + (1 - rᵢ)(1 - xᵢ)) modulo 2 for a given bit vector r = (r₀, …, rₙ₋₁).
+    // Inputs:
+    //      1) N qubits in arbitrary state |x⟩ (input register)
+    //      2) a qubit in arbitrary state |y⟩ (output qubit)
+    //      3) a bit vector of length N represented as Int[]
+    // The qubit array and the bit vector have the same length.
+    operation Oracle_ProductWithNegationFunction (x : Qubit[], y : Qubit, r : Int[]) : Unit {
+        
+        body (...) {
+            for (i in 0 .. Length(x) - 1) {
+                if (r[i] == 1) {
+                    CNOT(x[i], y);
+                } else {
+                    // do a 0-controlled NOT
+                    X(x[i]);
+                    CNOT(x[i], y);
+                    X(x[i]);
+                }
+            }
+        }
+        
+        adjoint invert;
+    }
+
+    // The algorithm: reconstruct the oracle in a single query
```

Comment:
Similarly here, if you could format this as a /// comment, that would be very helpful, thank you!